tests/unit/test_common.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
The coding statement is only needed if the file contains plan Unicode characters. We should not do that however, but always escape them as `\uXXXX` or `\UXXXXXXXX`.

Reply:
DONE